cmd/selfupdatectl/upload.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/fynelabs/selfupdate/cmd/selfupdatectl/internal/cloud"
+	"github.com/urfave/cli/v2"
+)
+
+type awsConfig struct {
+	endpoint   string
+	region     string
+	bucket     string
+	akid       string
+	secret     string
+	baseS3Path string
+}
+
+func s3upload() *cli.Command {
+	a := &application{}
+	config := &awsConfig{}
+
+	return &cli.Command{
+		Name:        "s3upload",
+		Usage:       "Upload multiple executable which has been properly signed to a S3 path",
```

Comment:
I think this doc is not right - it refers to multiple executables.
Also it could read as "properly signed" being the input, not the result - perhaps "Upload an executable file to S3, it will be signed and the signature uploaded too"?